tests/acceptance/features/coreApiWebdavPreviews/previews.feature

```diff
@@ -152,6 +152,21 @@ Feature: previews of files downloaded through the webdav API
       | new              | example.gif |
 
 
+  Scenario Outline: download previews of shared files inside of shared folder
+    Given using spaces DAV path
+    And user "Brian" has been created with default attributes and without skeleton files
+    And user "Alice" has uploaded file "filesForUpload/<resource>" to "/<resource>"
+    And user "Alice" has shared file "/<resource>" with user "Brian"
+    And user "Brian" has accepted share "/<resource>" offered by user "Alice"
+    When user "Brian" downloads the preview of shared resource "/<resource>" with width "32" and height "32" using the WebDAV API
```

Comment:
It would be confusing to use different steps for the same action. maybe use this new one for shared scenarios.